README.md

```diff
@@ -1,16 +1,51 @@
-# Release Integration Tests
+# Release Integration Test
 
+This README provides instructions on running a test using the provided shell script.
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## Test Steps and Expected Result
+
+1. **Pre-Running Test:** The following setup actions need to be completed before running the test script.
+   The test is running against RHTAP-Stage, so a user wants to monitior the test running throught UI or CLI 
+   will need an access to the cluster and be joined to the workspaces *release-dev-team* and *release-managed-team*. 
```

Comment:
`s/an access/access/`
`s/cluster and be joined to the/cluster and the workspaces/`